src/CheckoutSdk/Common/Four/AccountHolderType.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Runtime.Serialization;
+
+namespace Checkout.Common.Four
+{
+    public enum AccountHolderType
```

Comment:
We already have a `PaymentSenderType` class, let's refactor that class to use this one instead.